ledger/participant-integration-api/src/main/scala/platform/store/backend/postgresql/PostgresStorageBackend.scala

```diff
@@ -257,7 +257,7 @@ private[backend] object PostgresStorageBackend
     val pgSimpleDataSource = new PGSimpleDataSource()
     pgSimpleDataSource.setUrl(jdbcUrl)
 
-    Using.resource(pgSimpleDataSource.getConnection())(checkCompatibility)
+    Try(Using.resource(pgSimpleDataSource.getConnection())(checkCompatibility))
```

Comment:
as a consequence this won't do much on the first datasource initialisation if noconnectivity, but it will later as ds-s are created for dbdispatchers (at that time we already waited once for connectivity)